networks/known_networks.go

```diff
@@ -820,56 +820,84 @@ func MustGetSelectedNetworkConfig(networkCfg *config.NetworkConfig) []blockchain
 	if networkCfg == nil || len(networkCfg.SelectedNetworks) == 0 {
 		panic(fmt.Errorf("network config has no or empty selected networks. Use valid network(s) separated by comma from %v", getValidNetworkKeys()))
 	}
-	return MustSetNetworks(*networkCfg)
+	nets, err := MustSetNetworks(*networkCfg)
+	if err != nil {
+		panic(err)
+	}
+	return nets
 }
 
-func MustSetNetworks(networkCfg config.NetworkConfig) []blockchain.EVMNetwork {
+func MustSetNetworks(networkCfg config.NetworkConfig) ([]blockchain.EVMNetwork, error) {
```

Comment:
Please rename to `SetNetworks`, afaik we add `Must` prefix if the method returns valid result or panics